index.html

```diff
@@ -67,22 +64,55 @@ <h4 class="content__mission">We know how large objects will act, but things on a
 
     </section>
 
-<section class="strong-points">
-    <div class="strong-points___content">
-        <h2 class="content___reasons">WHY CHOOSE US
-        </h2>
-        <p>We know how large objects will act, but things on a <br>
-            small scale just do not act that way.<p>
-        
-            <div class="">
-            
-
-        </div>
-
+    <section class="strong-points">
+        <div class="container">
+            <div class="strong-points__content">
+                <h2 class="content__title">Why choose us</h2>
+                <p class="content__text">Problems trying to resolve the conflict between <br> the two major realms of
+                    Classical physics: Newtonian mechanics<p>
+            </div>
 
-</section>
+            <div class="grid-container">
+                <div class="grid-card">
+                    <h5 class="card__title"><img src="./img/svg/suitcase.svg" class="card-icon"
+                            alt="suitcase image">Digital Marketing</h5>
+                    <p class="card__text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog
+                    </p>
+                </div>
+
+                <div class="grid-card">
+                    <h5 class="card__title"><img src="./img/svg/shop.svg" class="card-icon" alt="shop image">Digital
+                        Marketing</h5>
+                    <p class="card__text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+                </div>
+
+                <div class="grid-card">
+                    <h5 class="card__title"><img src="./img/svg/notebook.svg" class="card-icon"
+                            alt="notebook image">Business Growing</h5>
+                    <p class="card__text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+                </div>
+
+                <div class="grid-card">
+                    <h5 class="card__title"><img src="./img/svg/clients.svg" class="card-icon"
+                            alt="clients image">National top 50 firms</h5>
+                    <p class="card__text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+                </div>
+
+                <div class="grid-card">
+                    <h5 class="card__title"><img src="./img/svg/branches.svg" class="card-icon"
+                            alt="branches image">National top 50 firms</h5>
+                    <p class="card__text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog </p>
+                </div>
+
+                <div class="grid-card">
+                    <h5 class="card__title"><img src="./img/svg/award.svg" class="card-icon" alt="award image">Web
+                        Development</h5>
+                    <p class="card__text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+                </div>
 
+            </div>
 
```

Comment:
@IvanCeban I added one more </div> here, but unfortunately (I`ve no idea why), it didn't upload.